addons/l10n_fi/__manifest__.py

```diff
@@ -0,0 +1,28 @@
+# coding=utf-8
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+# Copyright (C) RockIT Oy 2014-
+# Copyright (C) Avoin.Systems 2014-
+# Authors:
+#   Miku Laitinen (Avoin.Systems)
+#   Mikko Närjänen (Avoin.Systems)
+#   Toni Pesola (Avoin.Systems)
+#   Mikko Salmela (RockIT Oy)
+
+# noinspection PyStatementEffect
+{
+    "name": "Finland - Accounting",
+    "category": "Localization",
+    "version": "1.0.0",
+    "author": "RockIt Oy & Avoin.Systems",
+    "depends": [
+        "account",
+        "base_vat",
+        "base_iban",
+        "l10n_multilang",
```

Comment:
The point of l10n_multilang, is to translate accounting records like account and taxes name. Most countries do not need that as they have only one official language, they write their account name in their own language. For some countries like Belgium, Switzerland, Canada,.. the chart of account should be translated in and only in other official languages. For the other, we keep all the names in the source language only.